### ISSUE TITLE
Does buildifier support removing empty sections?

### ISSUE THREAD

**Issue by hengyang-zhao** [bazelbuild/buildtools#1086]
Does buildifier support removing empty sections?

For example

```
my_rule(
  name = "foo",
  deps = [
  ],
)
```

I would like to have deps removed.

Thanks!

**Comment by laurentlb** [comment]
No, it's not safe in the general case. It can modify the program execution (depending on the default value of the argument).

If you find this kind of cleanup useful, we could have a command in buildozer (`remove_if_empty`?).
